Add the option to save a pickle file (#159)
palvarezc/cavendish-particle-tracks: docs/user-manual.md

Once computed for the first time, the magnification parameters are stored and used to convert all measurements. If you need to recompute the magnification parameters, you can do so by clicking on the `Measure magnification` button again. The tool will remember previously computed magnification parameters, and will allow you to switch between them.

### Saving the data

> The data is stored internally as a list of `ParticleDecays` objects, which contain the information about the particles, their properties, as well as the magnification parameters.

Reviewer: ```suggestion
The data is stored internally as a list of `ParticleDecay` objects, which contain the information about the particles, their properties, as well as the magnification parameters.
```

Double-check that that links to the API? You might need to write `analysis.ParticleDecay` or something.

To save the data to a file for the analysis, click on the `Save to file` button. This will open a file dialog, which will allow you to select the file where you want to save the data. Two file types are suported: `CSV` and `Pickle`.

> The `CSV` file will save the data in a human-readable format, which can be opened in a text editor or a spreadsheet program. Make sure to import the data as a CSV file with the correct delimiter (`,`) and the right encoding (`UTF8`) so that the symbols are rendered correctly.

Reviewer: ```suggestion
`CSV` format is a readable comma-separated file format, which can be opened in a text editor, or a spreadsheet program, or using some other analysis tool like [pandas](https://pandas.pydata.org/). Make sure to import the data as a CSV file with the correct delimiter (`,`) and the correct encoding (`UTF8`) so that the symbols are rendered correctly.
```

> The `Pickle` file will save the data in a binary format, which can be open with Python using:

Reviewer: ```suggestion
A `Pickle` file is a binary format, which can be open with Python using:
```
